otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/Chart-template.yaml

```diff
@@ -21,4 +21,9 @@ version: ${IMAGE_TAG}
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "${IMAGE_TAG}"
\ No newline at end of file
+appVersion: "${IMAGE_TAG}"
+dependencies:
```

Comment:
this would deploy node exporter for our backdoor deployment/testing as well for AKS. May be have a flag to turn it off ?